mote/templates/aboutpage.html

```diff
@@ -0,0 +1,743 @@
+                                About
+                            </a>
+                        </li>
+                    </ul>
+                    <div class="d-flex flex-column justify-content-center align-items-center">
+                        <div class="input-group">
+                            <input type="text" autocomplete="off" class="form-control"
+                                placeholder="Search meeting logs"
+                                aria-label="Search meeting logs" aria-describedby="findtext"
+                                style="width: 350px;" id="findtext">
+                        </div>
+                        <div class="input-group" data-bs-auto-close="true">
+                            <ul class="dropdown-menu w-100" id="dropdown">
+                            </ul>
+                        </div>
+                    </div>
+                </div>
+            </div>
+        </nav>
+        <div class="container-lg">
+            <div class="card text-center mt-1">
+                <div class="card-body pt-0">
+                    <div class="text-center">
+                        <div class="mt-4">
+                            <h1 class="abouthead"><u>Mote 2.0</u></h1>
+
+                            <h4>Flask project used to aggregate and distribute IRC meeting minutes and logs for the Fedora Project</h4>
+                        
+                             <h2 class="abouthead"><u>Description</u></h2>
+                
+                            <p>
+                            Mote is a web application purposed to aggregate and distribute the minutes and logs for IRC/Matrix meetings of the Fedora Project community
+                            The project allows looking up meeting info with channel names, dates/times of occurrence and lists recent meetings for quick access
+                            The backend is written in Python with the use of libraries such as Flask micro web framework, BeautifulSoup4, Urllib3, Click, Werkzeug etc
+                            The frontend is written in HTML5, CSS3 and ES6 with the use of libraries such as Bootstrap 5, jQuery, Popper.JS, EasyQRCode etc
+                            </p>                    
+                            <h2 class="abouthead"><u>List of contributors for mote project</u></h2>
+                            <div class="row row-cols-1 row-cols-sm-2  row-cols-lg-3 row-cols-xl-4 g-4">
+                            <div class="col">
+                              <div class="card border-light bg-light mb-3">
+                                <div class="row g-0">
+                                  <div class="col-4">
+                                    <a href="https://github.com/cydrobolt">
+                                      <img src= "https://avatars.githubusercontent.com/u/1058150?v=4" style="width: 100px; height: 100px;" class="rounded-circle" alt="Chaoyi Zha">
```

Comment:
This image use the `rounded-circle` class while the others use `rounded-start`.
For consistency, you should choose one or another. Both looks good, so it's up to you to decide.

Reply:
sure